README.md

```diff
@@ -91,3 +92,40 @@ jobs:
            requirements_path: ".container/requirements.txt" 
            install_extras: "azure datadog"
 ```
+
+## build_helm_chart
+
+### Description
+
```

Comment:
Note here that action relies on git tags to be present in order to generate an artifact tag